src/transformers/models/auto/image_processing_auto.py

```diff
@@ -71,6 +71,7 @@
         ("idefics", "IdeficsImageProcessor"),
         ("imagegpt", "ImageGPTImageProcessor"),
         ("instructblip", "BlipImageProcessor"),
+        ("kosmos-2", "CLIPImageProcessor"),
```

Comment:
This is missed in `Kosmos-2` PR. 

We need this otherwise pipeline don't know what to do.